tests/sentry/api/endpoints/test_organization_metric_data.py

```diff
@@ -791,10 +796,10 @@ def test_orderby_percentile_with_many_fields_one_entity(self):
         are from the same entity
         """
         org_id = self.organization.id
-        metric_id_fcp = _indexer_record(org_id, TransactionMRI.MEASUREMENTS_FCP.value)
-        transaction_id = _indexer_record(org_id, "transaction")
-        transaction_1 = _indexer_record(org_id, "/foo/")
-        transaction_2 = _indexer_record(org_id, "/bar/")
+        metric_id_fcp = perf_indexer_record(org_id, TransactionMRI.MEASUREMENTS_FCP.value)
+        transaction_id = perf_indexer_record(org_id, "transaction")
+        transaction_1 = rh_indexer_record(org_id, "/foo/")
```

Comment:
This says transaction, why are you using `rh_indexer_record`

Reply:
same as in https://github.com/getsentry/sentry/pull/36308#discussion_r917724649